android/app/src/main/java/org/fossasia/openevent/common/utils/Utils.kt

```diff
@@ -48,10 +45,10 @@ object Utils {
         }
 
         @JvmStatic
-        fun checkStringEmpty(string: String): String {
+        fun checkStringEmpty(string: String?): String {
             var finalString = ""
             if (!isEmpty(string)) {
-                finalString = string
+                finalString = string ?: throw  NullPointerException()
```

Comment:
This was sarcasm. You obviously don't want to do this. !! does exactly this

Reply:
You could have used an emoji to make it look more like a sarcastic comment :laughing: 
sometimes it is difficult to figure out
but I guess I got your point now  :P